theme/highcontrast.less

```diff
@@ -877,6 +877,14 @@
         border: 1px solid @HCtextColor;
     }
 
+    #simulator .editor-sidebar:not(.topInstructions) {
+        border-right: 1px solid @HCtextColor;
+    }
+
+    #simulator .topInstructionsWrapper .editor-sidebar {
```

Comment:
> In this case, #simulator is unnecessary anyway, so I just removed it.

Did you leave `#simulator` in by accident?

Reply:
In this case it's helpful to add specificity to the selector. I only needed to remove it in a couple of spots in tutorial-sidebar.less where it was throwing off the selector ordering.

Reply:
Sorry, PR description was confusing :)

I noticed the selector-ordering issue while looking into high contrast mode, but it's actually unrelated to the HC changes. It was related to the border when *not* in high contrast mode.